client/web/src/SourcegraphWebApp.tsx

```diff
@@ -461,6 +466,9 @@ class ColdSourcegraphWebApp extends React.Component<SourcegraphWebAppProps, Sour
                                     fetchRecentSearches={fetchRecentSearches}
                                     fetchRecentFileViews={fetchRecentFileViews}
                                     fetchUserCodeMonitors={listUserCodeMonitors}
```

Comment:
Should we rename this property to match the name of the function?
```suggestion
                                    listUserCodeMonitors={listUserCodeMonitors}
```

Reply:
I'll change the name of the function to match the prop name